data/working_groups.yml

```diff
@@ -0,0 +1,74 @@
+---
+# template
+# - title:
+#   goal:
+#   open_letter:
+#   communication_channel:
+#   url:
+#   chairs:
+#     - name:
+#       email:
+#     - name:
+#       email:
+#   status:
+#   outcome:
+#       - url:
+#       - url:
+#
+
+# ACTIVE
+
+-   title: BIDS 2.0
+    goal: Discussions and suggestions of backwards incompatible changes to BIDS
+    open_letter: TBD
+    communication_channel: TBD
+    url: https://github.com/bids-standard/bids-2-devel
+    chairs:
+    -   name: Yaroslav Halchenko
```

Comment:
@yarikoptic  Can you give me some links for the open letter, a communication channel where people can join and interact with the working group 

can you cross check that the rest matches the BIDS 2.0 working group

Reply:
commented this out for now until I hear more from an official announcement for a working group for BIDS 2.0